README.md

```diff
@@ -129,6 +131,19 @@ Solomon proposed to support VXLAN from the Docker core.
 - Docker Hub Enterprise
 - Private Docker registries using the “registry" docker image.
 
+## Configuration Management Tools with Docker support 
+
+- Ansible 
+- Saltstack
+- Chef 
+- Puppet 
+- Packer
+
+## Networking with Docker
+
+- [Skydock](https://github.com/crosbymichael/skydock) 
```

Comment:
Skydock seems like a service discovery tool. Put it into the "Service discovery & configuration" section instead?